styles.css

```diff
@@ -3,150 +3,222 @@
   margin: 0;
   padding: 0;
   box-sizing: border-box;
-  font-size:18px; /*this is for accessibility for dyslexic people*/
-  font-family:sans-serif;
+  font-size: 18px; /*this is for accessibility for dyslexic people*/
+  font-family: sans-serif;
 }
 
-:root{
-  --color-base:#C2B9B0;
-  --color-highlight:#E7717D;
-  --color-dark:#7E685A;
-  --width-sm:20rem;
-  --width-md:40rem;
-  --width-lg:60rem;
+:root {
+  --color-base: #f5f5f5;
+  --color-highlight: #e7717d;
+  --color-dark: #7e685a;
+  --color-black: #000000;
+  --color-white: #ffffff;
+  --width-sm: 20rem;
+  --width-md: 40rem;
+  --width-lg: 60rem;
 }
 
-.center{
-  margin-left:auto;
-  margin-right:auto;
-  width:80%
+html {
+  scroll-behavior: smooth;
 }
 
-.row{
-  display:flex;
-  flex-flow: row wrap;
-  align-items:center;
-  justify-content: space-around;
+body {
+  background-color: var(--color-base);
+}
+
+.center {
+  margin-left: auto;
+  margin-right: auto;
+  width: 80%;
 }
 
-.align-start{
-  align-items:flex-start;
+.flex {
+  display: flex;
 }
 
-.align-end{
-  align-items:flex-end;
+.row {
```

Comment:
The page is quite simple so I'm not sure we will actually need that much control with so many classes, but I see what you are going for. Let's keep these changes and see how they work for us.

Reply:
Yeah, we can always go back and delete extra stuff, but refactoring later on to add stuff will be difficult